source/engineering/abaqus/troubleshooting.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Troubleshooting"
+---
+
+
+## License not Authorized for User
+
+```text
+Failed to checkout DSLS license for: QAE
+ERROR 1A000035: License not authorized for this user
+...
+```
+
+If you receive this error when trying to run Abaqus, then you first need to
+register for a university Abaqus license
+[here](https://forms.office.com/pages/responsepage.aspx?id=MH_ksn3NTkql2rGM8aQVG0Ep7WjCs19BiAIhW6hXMWlUREtGTFQwS0sySUtGNzZON1MzR1A5TUJKMS4u).
+
+
+## Out of Memory
+
+If you do not request sufficient memory for your job, then it may be terminated by
+the job scheduler with the following error message (check the log output files):
+
+
+```text
+slurmstepd: error: Detect 1 oom-kill event(s) in StepId=xxxxxx.batch. Some of your processes may have been killed by the cgroup out-of-memory handler.
+```
+
+This may also be accompanied by the following Abaqus error message (or similar):
+
+```text
+Abaqus Error: The executable explicit_dp
+aborted with system error "Termination signal" (signal 9)
+```
+
+You can address this issue by increasing the requested `--mem-per-cpu` in your job submission script.
+
+
+## Node Configuration not Available
+
+```text
+sbatch: error: Batch job submission failed: Requested node configuration is not available
+```
+
+This error message can occur when you request too many resources (_e.g._ cpus,memory) than is available
+on the cluster nodes.
+
+```{note}
+Your total memory request is the `--mem-per-cpu` parameter multiplied by
+the number of cpus you have requested.
+On BlueCrystal you can request up to a total of 132GB RAM per node.

```

Comment:
It is reasonable to request all physical memory on a node? I'm not sure -- do we need to leave room for the OS? I usually use `--mem=0` to request all memory available per node (and let the scheduler decide how much this is), rather than trying to specify.

Either way, I would suggest removing specific information about the memory per node and instead linking to a page in the ACRC HPC user docs with information about the hardware of the clusters (e.g. https://www.acrc.bris.ac.uk/protected/hpc-docs/training/intro-to-hpc-slurm/acrc_systems.html). Then this information does not need to be maintained in two places.

Reply:
Good point. I didn't know you could do `--mem=0` to request all available!
Yeah I agree, linking to the ACRC docs is a better idea. (I couldn't find this specs page originally, I was expecting this information to be at the top-level maybe under 'HPC Systems Overview'?)